src/helper/selection-tools/move-tool.js

```diff
@@ -153,16 +158,43 @@ class MoveTool {
             } else {
                 item.position = item.data.origPos.add(dragVector);
             }
+
+            if (bounds) {
+                bounds = bounds.unite(item.bounds);
+            } else {
+                bounds = item.bounds;
+            }
         }
         
         if (this.firstDrag) {
             // Show the center crosshair above the selected item while dragging.
             getDragCrosshairLayer().visible = true;
             this.firstDrag = false;
         }
-        const opacity = Math.max(0,
-            1 - ((CENTER.getDistance(this.selectionCenter.add(dragVector)) / CENTER.x) * (4 * paper.view.zoom)));
-        getDragCrosshairLayer().opacity = opacity;
+
+        let opacityMultiplier = 1;
+        const newCenter = this.selectionCenter.add(dragVector);
```

Comment:
Ooof this really needs some documentation explaining what the intention of this is.

Reply:
Added comments